### PR TITLE
manage-fluent-repositories.sh: Further automation of creating RPM repo

### PR DESCRIPTION
**Pull request by ashie**
In addition, fix wrong usage in help.

Signed-off-by: Takuro Ashie <ashie@clear-code.com>

### CODE REVIEW

File: td-agent/manage-fluent-repositories.sh [hunk 1]
```diff
@@ -122,13 +122,16 @@ EOF
...
-	# sign rpm repository
-	find $FLUENT_RELEASE_DIR/4 -name 'repomd.xml'
-	for f in `find $FLUENT_RELEASE_DIR/4 -name 'repomd.xml'`; do
-	    if [ -f "$f.asc" ]; then
-		rm -f $f.asc
+	# update & sign rpm repository
+	repodirs=`find "${FLUENT_RELEASE_DIR}" -regex "^${FLUENT_RELEASE_DIR}/4/\(redhat\|amazon\)/[2789]$"`
+	for repodir in $repodirs; do
+	    createrepo_c -v "${repodir}"
+
```
Comment (kenhys) [inline td-agent/manage-fluent-repositories.sh:129]: In the previous versions, repodata/repomd.xml is generated under each x86_64 or aarch64.
Is it intentional change?, If so and it should be changed, it's ok for me.

Comment (kenhys) [reply td-agent/manage-fluent-repositories.sh:129]: I mean: 

Before: td-agent-release/4/redhat/8/x86_64/repodata/repomd.xml
After:  td-agent-release/4/redhat/8/repodata/repomd.xml

Comment (ashie) [reply td-agent/manage-fluent-repositories.sh:129]: It's not intended. I'll fix it.

Comment (ashie) [reply td-agent/manage-fluent-repositories.sh:129]: Fixed in 63c1e084fe324c1c90a8c7686107710cddc8d672

**Pull request merged**